sidecar/sidecar.go

```diff
@@ -77,6 +77,12 @@ func sidecarCommand(context *cli.Context) error {
 	if err != nil {
 		return err
 	}
+	if conf.Proxy {
+		err = conf.GenProxyConfig()
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
I'm curious where the envoy config is being generated? If that's in `Run` then the nginx config should be generated there also. 

But given that this is the 1st dynamic generation of nginx config it seems to me to be a subjective/philosphical call: 
- Is proxy config generation part of the part of pre-Run config processing? 
- Or is it part of the Run phase?

Functionally the outcome seems identical.